src/schemas/json/jsconfig.json

```diff
@@ -111,8 +111,8 @@
               "default": false
             },
             "jsx": {
-              "description": "When down-level compiling, specify JSX code generation: 'preserve', 'react', or 'react-native'. Requires TypeScript version 2.2 or later.",
```

Comment:
I remove the `Requires TypeScript` part of the description since it is not present in the tsconfig schema